#! /usr/bin/env python

PACKAGE='controllers'

import roslib;
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#       	Name		Type      	Level	Description			Default 	Min	Max 	Values
gen.add(	"t_fail_max", 	int_t,		0,	"Maximum time fail motor",	25,		0,	50)
gen.add(	"Fail_motor", 	bool_t,		0,	"Motor Failure flag",		False)
gen.add(	"Motor_1", 	bool_t,		0,	"Motor Number 1",		False)
gen.add(	"Motor_2", 	bool_t,		0,	"Motor Number 2",		False)
gen.add(	"Motor_3", 	bool_t,		0,	"Motor Number 3",		False)
gen.add(	"Motor_4", 	bool_t,		0,	"Motor Number 4",		False)
gen.add(	"Motor_5", 	bool_t,		0,	"Motor Number 5",		False)
gen.add(	"Motor_6", 	bool_t,		0,	"Motor Number 6",		False)

exit(gen.generate(PACKAGE, "controllers", "setMotorFailure"))
